dashboard/api_test.py

```diff
@@ -22,8 +22,10 @@
 )
 from dashboard.utils import MMTrack
 from micromasters.factories import UserFactory
-from micromasters.utils import load_json_from_file
+from micromasters.utils import load_json_from_file, is_subset_dict
 from search.base import ESTestCase
+from seed_data.lib import ensure_cached_edx_data_for_user, set_course_run_past
+from seed_data.management.commands.alter_data import set_course_to_failed, add_past_failed_run
```

Comment:
see [this comment](https://github.com/mitodl/micromasters/pull/1740#discussion_r88065190)